src/styles/index.css

```diff
@@ -0,0 +1,3 @@
+@import "./reset.css";
+@import "./blocks/stars.css";
+@import "./blocks/card.css";
```

Comment:
Also would looks better if you add some spaces between you `@inputs` and maybe some info about it) just an advice)